File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.java
class Solution {
    public int minBitFlips(int start, int goal) {
        int result = 0;

        int xor = start ^ goal;

        while(xor!=0){
            result = result + (xor & 1);
            xor = xor >> 1;
        }
        return result;
    }
}

File: 2220-minimum-bit-flips-to-convert-number/2220-minimum-bit-flips-to-convert-number.cpp
class Solution {
public:
    int minBitFlips(int start, int goal) {
        int result = 0;

        int XOR = start ^ goal;

        while(XOR!=0){
            result = result + (XOR & 1);
            XOR = XOR >> 1;
        }

        return result;
    }
};